stats/stats/src/lib.rs

```diff
@@ -1,14 +1,216 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+use chrono::NaiveDateTime;
+use entity::chart_data_int;
+use sea_orm::{
+    DatabaseConnection, DbBackend, DbErr, EntityTrait, FromQueryResult, JsonValue, Statement,
+};
+use stats_proto::blockscout::stats::v1::{ChartInt, Counters, PointInt};
+use std::collections::HashMap;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ReadError {
+    #[error("database error {0}")]
+    DB(#[from] DbErr),
+    #[error("chart {0} not found")]
+    NotFound(String),
+}
+
+fn parse_json(row: JsonValue) -> Option<(String, (String, u64))> {
+    let name = row.get("name")?.as_str()?;
+    let date = row.get("date")?.as_str()?;
+    let value = row.get("value")?.as_u64()?;
+    Some((name.into(), (date.into(), value)))
+}
+
+fn get_counter_from_data(
+    data: &HashMap<String, (String, u64)>,
+    name: &str,
+) -> Result<u64, ReadError> {
+    data.get(name)
+        .map(|(_, value)| *value)
+        .ok_or_else(|| ReadError::NotFound(name.into()))
+}
+
+pub async fn get_counters(db: &DatabaseConnection) -> Result<Counters, ReadError> {
+    let data = JsonValue::find_by_statement(Statement::from_string(
```

Comment:
I would suggest to use additional custom model instead of JsonValue:
```rust
#[derive(Debug, FromQueryResult)]
pub struct ChartData {
    name: ...,
    date: ...,
     value: ...,
}
```
https://www.sea-ql.org/SeaORM/docs/basic-crud/raw-sql/